grocerylist.py

```diff
@@ -68,20 +70,35 @@ def user_lst():
             food_lst.append(item)
     return food_lst
 
+def save_lst():
+#If no existing list, then save list
+#If existing list, prompt user to save as new list or replace existing list?
 
-# save = input('Would you like to save this list? Y/N: ')
-# if save == 'Y':
-#   fp.writelines(food_lst)
+#Prompt user to save list
+    save = input('Would you like to save this list? Y/N: ')
+#If user selects to save new list
+    if save == 'Y':
+        fp = open(r'C:\Users\Pookie\Downloads\python\grocery_list.txt', 'w')
+#Save each item in list on a new line in .txt
+        for item in food_lst:
+            fp.write("%s\n" % item)
+        print(f"List Saved")
+#If user selects not to save their list
+    elif save == 'N':
+        print("List erased")
 
 
-def item_count(item_cnt):
-    pass
-
+def item_count():
+    item_cnt = len(food_lst)
+    return item_cnt
 
 
 def main():
     existing_lst()
     groceries = food_lst
```

Comment:
Don't need to re-assign here if you'll use the var to print it out below. Could chop this line out and just print out food_lst